ui/raidboss/data/06-ew/dungeon/lapis_manalis.ts

```diff
@@ -0,0 +1,206 @@
+import Conditions from '../../../../../resources/conditions';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.LapisManalis,
+  timelineFile: 'lapis_manalis.txt',
+  timelineTriggers: [
+    {
+      id: 'Lapis Manalis Cagnazzo Tsunami',
+      regex: /Tsunami/,
+      beforeSeconds: 5,
+      response: Responses.bigAoe(),
+    },
+  ],
+  triggers: [
+    // ---------------- first trash ----------------
+    {
+      id: 'Lapis Manalis Albus Griffin Winds of Winter',
+      type: 'StartsUsing',
+      netRegex: { id: '8011', source: 'Albus Griffin', capture: false },
+      response: Responses.aoe(),
+    },
+    // ---------------- Albion ----------------
+    {
+      id: 'Lapis Manalis Albion Call of the Mountain',
+      type: 'Ability',
+      netRegex: { id: '7A7C', source: 'Albion', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Avoid stampeding animals',
+        },
+      },
+    },
+    {
+      id: 'Lapis Manalis Albion Albion\'s Embrace',
+      type: 'StartsUsing',
+      netRegex: { id: '7A85', source: 'Albion' },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Lapis Manalis Albion Right Slam',
+      type: 'StartsUsing',
+      netRegex: { id: '802D', source: 'Albion', capture: false },
+      response: Responses.goLeft(),
+    },
+    {
+      id: 'Lapis Manalis Albion Left Slam',
+      type: 'StartsUsing',
+      netRegex: { id: '802E', source: 'Albion', capture: false },
+      response: Responses.goRight(),
+    },
+    {
+      id: 'Lapis Manalis Albion Icy Throes',
+      type: 'StartsUsing',
+      netRegex: { id: '7A83', source: 'Albion' },
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Lapis Manalis Albion Roar of Albion',
+      type: 'StartsUsing',
+      netRegex: { id: '7A84', source: 'Albion', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Hide Behind Boulder',
+          de: 'Hinter Felsen verstecken',
+          fr: 'Cachez-vous derrière le rocher',
+          ja: 'ボルダーの後ろに',
+          cn: '躲在石头后',
+          ko: '돌 뒤에 숨기',
+        },
+      },
+    },
+    // ---------------- Galatea Magna ----------------
+    {
+      id: 'Lapis Manalis Galatea Magna Waxing Cycle',
+      type: 'StartsUsing',
+      netRegex: { id: '7A9[13]', source: 'Galatea Magna', capture: false },
```

Comment:
Are there different ability ids here or is one of these the damage and the other self-targeted? (If so, maybe just sync on self-targeted?)

Reply:
It's a two-hit combo, with separate abilities that have different cast durations.  `7A91` is the first hit (chariot), `7A93` is the second (dynamo).  I triggered on both just because they start at the same time and this way we catch which ever hits the log first.  It's not strictly necessary.